### PR TITLE
[zh] Sync concepts pages for architecture nodes and controller

### PR DESCRIPTION
**Pull request by ydFu**
* Update architecture\controller.md and architecture\node.md

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh/docs/concepts/architecture/nodes.md [hunk 1]
```diff
@@ -30,9 +31,10 @@ The [components](/docs/concepts/overview/components/#node-components) on a node
 {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}.
 -->
 Kubernetes 通过将容器放入在节点（Node）上运行的 Pod 中来执行你的工作负载。
-节点可以是一个虚拟机或者物理机器，取决于所在的集群配置。每个节点都包含用于运行
-{{< glossary_tooltip text="Pod" term_id="pod" >}} 所需要的服务，这些服务由
-{{< glossary_tooltip text="控制面" term_id="control-plane" >}}负责管理。
+节点可以是一个虚拟机或者物理机器，取决于所在的集群配置。每个节点由
+{{< glossary_tooltip text="控制面" term_id="control-plane" >}} 负责管理，
+并包含所需的服务进而运行
```
Comment (tengqm) [inline content/zh/docs/concepts/architecture/nodes.md:36]:
```suggestion
并包含运行 {{< glossary_tooltip text="Pods" term_id="pod" >}} 所需的服务。
```

Comment (ydFu) [reply content/zh/docs/concepts/architecture/nodes.md]: Thank you for suggesting me.

**Review by tanjunchen — COMMENTED** [review]
/approve

**Pull request merged**